core/tests/ui/e2e/register.spec.ts

```diff
@@ -35,5 +35,5 @@ test('Account is not created if email is already in use', async ({ page, account
 
   await page.getByRole('button', { name: 'Create account' }).click();
 
-  await expect(page.getByText('The email address is already in use.')).toBeVisible();
```

Comment:
I just had to fix an issue in the wordpress issue, but this is a security concern allowing bad actors find out if email addresses are valid. Changed it over to a more generic error.